stable/jenkins/README.md

```diff
@@ -128,29 +128,69 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `fullnameOverride`                | Override the full resource names     | `jenkins-{release-name}` (or `jenkins` if release-name is `jenkins`) |
 | `namespaceOverride`               | Override the deployment namespace    | Not set (`Release.Namespace`)             |
 | `master.componentName`            | Jenkins master name                  | `jenkins-master`                          |
-| `master.image`                    | Master image name                    | `jenkins/jenkins`                         |
-| `master.tag`                      | Master image tag                     | `lts`                                     |
-| `master.imagePullPolicy`          | Master image pull policy             | `Always`                                  |
-| `master.imagePullSecretName`      | Master image pull secret             | Not set                                   |
+| `master.enableXmlConfig`          | enables configuration done via XML files | `false`                               |
+| `master.JCasC.enabled`            | Whether Jenkins Configuration as Code is enabled or not | `true`                  |
+| `master.JCasC.defaultConfig`      | Enables default Jenkins configuration via configuration as code plugin | `true`  |
+| `master.sidecars.configAutoReload` | Jenkins Config as Code auto-reload settings |                                   |
+| `master.sidecars.configAutoReload.enabled` | Jenkins Config as Code auto-reload settings (Attention: rbac needs to be enabled otherwise the sidecar can't read the config map) | `true`                                                      |
+| `master.sidecars.configAutoReload.image` | Image which triggers the reload | `kiwigrid/k8s-sidecar:0.1.144`           |
+| `master.sidecars.configAutoReload.env` | Environment variables for the Jenkins Config as Code auto-reload container  | Not set |
+| `master.testEnabled`              | Can be used to disable rendering test resources when using helm template | `true`                         |
+| Jenkins Configuration Files & Scripts |                                  |                                           |
+| `master.JCasC.configScripts`      | List of Jenkins Config as Code scripts | `{}`                                    |
+| `master.initScripts`              | List of Jenkins init scripts         | `[]`                                      |
+| `master.overwriteConfig`          | Replace init scripts and config w/ ConfigMap on boot  | `false`                  |
+| `master.credentialsXmlSecret`     | Kubernetes secret that contains a 'credentials.xml' file | Not set               |
+| `master.secretsFilesSecret`       | Kubernetes secret that contains 'secrets' files | Not set                        |
```

Comment:
Should we deprecate those?

Reply:
We can definitely start deprecating fields that have a JCasC alternative.